sessions/state.go

```diff
@@ -117,30 +145,40 @@ func CreateState(r *http.Request, w http.ResponseWriter,
 // Finally, it returns a State struct, which contains information associated
 // with the particular OIDC flow.
 func VerifyState(r *http.Request, w http.ResponseWriter,
-	store sessions.Store) (*State, error) {
+	store sessions.Store, dynamicOidcStateCookieName bool) (*State, error) {
 
 	// Get the state from the HTTP param.
 	var stateParam = r.FormValue("state")
 	if len(stateParam) == 0 {
 		return nil, errors.New("Missing url parameter: state")
 	}
 
+	oidcStateCookieName := oidcStateCookie
+	stateValue := stateParam
+	nonce := ""
+	if(dynamicOidcStateCookieName) {
```

Comment:
looks like you're missing gofmt here, if should have space before the param

Reply:
thank you fixed. For some reason the linter config in my vscode broke and I haven't had the cycles to fix it (it either doesn't lint, or lints in a way that somehow changes the whole file. I will fix it at some point.